src/mainwindow.cpp

```diff
@@ -2189,8 +2190,8 @@ void MainWindow::maximizeWindow()
 void MainWindow::minimizeWindow()
 {
 #if defined(Q_OS_LINUX) || defined(Q_OS_FREEBSD)
```

Comment:
In other places we're now using `#if defined(Q_OS_UNIX) && !defined(Q_OS_MACOS)`, and I think it would be good to change it here too.

Reply:
Absolutely! I just deliberately chose not to do that in this PR to avoid making it too overwhelming for reviewers. :p

As soon as this and other PRs I have are accepted (or rejected :p), I'll send a new one updating those as well. :D